content/blog/2019/announcing-1.0-eol/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Support for Istio 1.0 ends on June 19th, 2019
+description: Upcoming Istio 1.0 EOL announcement
+publishdate: 2019-05-20
+attribution: The Istio Team
+release: 1.0
+---
+
+According to Istio's [support policy](https://istio.io/about/release-cadence/), LTS releases like 1.0 are supported for three months after the next LTS release.   Since [1.1 was released on March 19th](https://istio.io/about/notes/1.1/), support for 1.0 will end on June 19th, 2019.
+
+At that point we will stop backporting fixes for security issues and critical bugs to 1.0, so we strongly encourage you to upgrade to 1.1 (currently 1.1.7).  If you don't do this you may put yourself in the position of having to do a major upgrade on a short timeframe to pick up a critical fix.   
```

Comment:
Can I make the current version string (1.1.7) dynamic?

Reply:
Yes, how about just saying:

"...so we encourage you to upgrade to the latest version of Istio ({{<istio_release_name>}}).